File: Level1/23.정수내림차순으로배치하기/23.정수내림차순으로배치하기.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

long long solution(long long n) {
    long long answer = 0;
    vector<int> arr;
    while(n>0){
        arr.push_back(n%10);
        n = n/10;
    }
    sort(arr.begin(), arr.end());
    for(int i = 0 ; i<arr.size() ; i++){
        answer +=pow(10,i) * arr[i];
    }
    return answer;
}

File: Level1/23.정수내림차순으로배치하기/23.정수내림차순으로배치하기.java
import java.util.ArrayList;
import java.util.Collections;
class Solution {
    public long solution(long n) {
        long answer = 0;
        ArrayList<Integer> arr = new ArrayList<Integer>();
        while(n>0){
            arr.add((int)(n%10));
            n = n/10;
        }
        Collections.sort(arr);
        for(int i = 0 ; i<arr.size() ; i++){
            answer += Math.pow(10,i) * arr.get(i);
        }
        return answer;
    }
}